core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1930,14 +1930,40 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  /**
+   * @deprecated use {@link #MASTER_INODE_METASTORE #MASTER_BLOCK_METASTORE} instead
```

Comment:
```suggestion
   * @deprecated use {@link #MASTER_INODE_METASTORE} and {@link #MASTER_BLOCK_METASTORE} instead
```

Reply:
@maobaolong Done, thanks for this suggestion!